saq/job.py

```diff
@@ -129,6 +129,23 @@ class Job:
     status: Status = Status.NEW
     meta: dict[t.Any, t.Any] = dataclasses.field(default_factory=dict)
 
+    def __str__(self) -> str:
```

Comment:
why not have this just be the same as repr?

Reply:
__repr__ now just calls info(True)